extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -250,28 +250,30 @@ export class Bind {
 
   /**
    * Merges `state` into the current state and immediately triggers an
-   * evaluation unless `opt_skipEval` is false.
+   * evaluation unless `skipEval` is false.
    * @param {!JsonObject} state
-   * @param {boolean=} opt_skipEval
-   * @param {boolean=} opt_skipAmpState
+   *
+   * @param {Object} opts options bag
+   * @param {boolean=} opts.skipEval
+   * @param {boolean=} opts.skipAmpState
```

Comment:
Is our linter/compiler cool with this? Might need to do:
```js
@param {{skipEval: (boolean|undefined), skipAmpState: (boolean|undefined)}=} opts
```

Reply:
Also probably need to extract the typedef make this an extern in `amp.extern.js`. It'll prevent property renaming since this API will be used across binaries.

Reply:
> Is our linter/compiler cool with this? Might need to do:

Yep! See [parameters with properties](https://jsdoc.app/tags-param.html#parameters-with-properties). I didn't know about the syntax you just wrote though, its super concise.

Reply:
(and I've extracted the typedef)